components/UploadedItems.js

```diff
@@ -1,57 +1,47 @@
-import React, { Component } from "react";
+import React, { useState, useEffect } from "react";
 import Link from "next/link";
-
 const url = `https://archive.org/advancedsearch.php?q=bub.wikimedia+&rows=0&output=json`;
 
-class UploadedItems extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      booksUploaded: 0
-    }
-    this.fetchData = this.fetchData.bind(this);
-  }
-
-  fetchData() {
-    fetch(url)
-      .then((response) => response.json())
-      .then((responseJson) => {
-        const booksCount = responseJson.response.numFound;
-        this.setState({ booksUploaded: booksCount });
-      })
-      .catch((error) => {
-        console.error(error);
-      });
-  }
+const UploadedItems = () => {
+  const [booksUploaded, setBooksUploaded] = useState(0);
 
-  componentDidMount() {
-    this.fetchData();
-  }
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(url);
+        const responseJson = await response.json();
+        const booksCount = await responseJson.response.numFound;
+        console.log("booksCount", booksCount);
```

Comment:
This logging statement can be removed I believe

Reply:
Noted! I removed the console log and added the .env.example file back.